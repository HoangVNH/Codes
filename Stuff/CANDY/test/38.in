10
26
XXXDDDDVVVVVDDDDDDDDVVVDDD
41
VVVVVVVXDDXXXXDDDDDDDDDVVVVVVVVDDDVVXXXVV
33
DDDDDVVVXXXXXXXXXXXXXDXXXXXXXXXDD
48
DDDDDDDDDDDDVVVXXDDDDDDDDXXXXXXXXXVVVVXXVVVVVVVV
20
DDDDVVVVVXVVVVVVVVVV
28
DDVDDXXXXXXXXXXXVVVVVVVVDDDX
22
DDDDVVVVVVVVDVVDDDDXXX
49
VVVVDDDDDXXXXVVVVVDVVVVXXXXDDDDDDDDXXXXDDXXXXXXVX
42
VVVVVVVVXXXXVDDDDDDDVVDDDDVVVVXXDXXDDDVVVV
23
VVVVVVVVXXXVVVXXXDDDDDV
