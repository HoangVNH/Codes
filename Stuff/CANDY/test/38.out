25
40
32
47
19
27
21
48
41
22
